src/psij/testing/service.py

```diff
@@ -405,7 +406,7 @@ def getSite(self, site_id, tests_to_match):
         run_id = ""
 
         try:
-            run_id = RunEnv.objects(site_id=site_id).order_by('-test_start_time')[0].run_id
+            run_id = RunEnv.objects(site_id=site_id,branch='master').order_by('-run_start_time')[0].run_id
```

Comment:
For PSI/J, it's "main" not "master". At some point github switched to having "main" as the default. The SDK repo was created before that point and the PSI/J one after. You could do something like `branch__in = ['main', 'master']` or, better, have the branch filter as a parameter: if a list is specified, filter by that branch, if not, return all branches.